commcare_export/version.py

```diff
@@ -17,6 +17,9 @@ def stored_version():
 
 
 def git_version():
+    if os.environ.get('DET_EXECUTABLE'):
```

Comment:
This was added so that running the binary will not output an error saying "git was not found". It's expected that folks using the binary might not have git installed so deducing the version from git as is done in this function is redundant for the bundled environment.

Reply:
This shouldn't be an issue for built versions of the tools since we write the version to a file and use that in preference to the git version.